lang/typescript/typescript.py

```diff
@@ -18,6 +18,16 @@
 
 @ctx.action_class("user")
 class UserActions:
+    def code_default_function(text: str):
```

Comment:
Why not define this one in `javascript.py`? Then typescript gets it for free

Reply:
Javascript gets the default implementation that passes through to code_private_function anyway.

Reply:
Sorry–not following.  Why do we need this for typescript but not javascript?

Reply:
user.code_default_function's default implementation delegates to user.code_private_function. In javascript.py's code_private_function outputs `function myFunc` which is the desired behaviour. In typescript.py code_private_function outputs `private function myFunc`. This means in typescript there was previously no way of outputting a function without modifiers (which is what you'd want to do outside a class context).

Reply:
I guess I'm not seeing why do we want `code_private_function` to do one thing in javascript and another thing in typescript